proco/locations/migrations/0011_auto_20201130_0908.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 2.2.15 on 2020-11-30 09:08
+
+from django.db import migrations
+
+
+def set_not_verified_statuses(apps, schema_editor):
+    Country = apps.get_model('locations', 'Country')
+
+    for country in Country.objects.filter(data_source__icontains='osm'):
```

Comment:
change status only for countries having no connectivity/coverrage data. or if you want to use data source, you can check countries having ONLY osm in data source